PlatformConfigurations/IEEE802.1xPortBasedAuthentication.md

```diff
@@ -0,0 +1,872 @@
+---
+title: IEEE 802.1x (Port Based Authentication)
+parent: Platform Configuration
+nav_order: 5
+layout: default
+---
+
+# IEEE 802.1x (Port Based Authentication)
+
+## Introduction
+
+In this guide, we will quickly explain what IEEE 802.1x (Port Based Authentication) is
+and give an example implementing it.
+
+IEEE 802.1x is a standard for port-based network access control that provides an authentication
+mechanism to devices wishing to join a LAN or WLAN. IEEE 802.1x uses the Extensible Authentication Protocol
+(EAP) to transmit authentication information over ethernet frameworks. The three main components
+include the **Supplicant, Authenticator, and Authentication Server**.
+
+- **Supplicant**: The Supplicant is the device attempting to access the network. It is often the one that
+  participates in the EAP transaction with the Authenticator.
+
+- **Authenticator**: The Authenticator is the network device that connects the supplicant to the network. It acts as the
+  access point for the Supplicant and controls the type of traffic flow between the Supplicant and
+  the network before and after authentication. The Authenticator sends
+  EAP requests to the Supplicant to identify a connecting device.
+  In IEEE 802.1x, this is also more broadly refered to as a Network Access Server.
+
+- **Authentication Server**: The Authentication Server receives requests for network access
+  and verifies devices attempting to connect to the network. This information is then passed back to the authenticator
+  where traffic flow is changed based on the results. This is also commonly refered to as a RADIUS Server.
+  It is also often known as an AAA (Authentication, Authorization, and Accounting) Server.
+
+The image below is a good visualization of the 802.1x Progression.
+
+![Seqeunce Diagram of the 802.1x Progression](../Images/ImagesForPlatformConfiguration/802.1x_Progression.png)
+
+In this document, we will be using the following free publicly available software:
```

Comment:
Why are we using that thing called freeradius? )i.e say we'll use it as the "authentication server").

Reply:
I will include a paraph explaining what each utility does underneath this line.
![image](https://github.com/user-attachments/assets/33aa97fa-b91b-4721-b7b8-6480d20415b6)